.github/workflows/bump-version.yml

```diff
@@ -33,6 +32,7 @@ jobs:
         with:
           ref: ${{ github.ref_name }}
           token: ${{ secrets.CLOUDSEC_MACHINE_TOKEN }}
+          fetch-depth: 0
```

Comment:
needed to get all branches as we'll be extracting the `latest_major_latest_minor` and `previous_major_latest_minor`